OpenRA.Mods.Common/Traits/Voiced.cs

```diff
@@ -32,22 +30,23 @@ public class Voiced : IVoiced
 	{
 		public readonly VoicedInfo Info;
 
+		public string VoiceSet { get; }
```

Comment:
Because it implements it from the interface, and I don't want to mess with that more than I need to here.